### ISSUE TITLE
arm64 macOS ("Apple Silicon", "M1") install friction

### ISSUE THREAD

**Issue by spencerwilson** [aws/aws-elastic-beanstalk-cli-setup#103]
**1. Please specify the following:**
 - OS: [e.g. OS X, Ubuntu, Windows, etc.]: macOS Big Sur 11.0.1
 - Shell (Bash, PowerShell, Zsh, etc.): Zsh

**2. Description**
This is informational, for those looking to run `eb` on their ARM Macs.

1. Python 3.7.2 doesn't build under macOS arm64. Update [this line](https://github.com/aws/aws-elastic-beanstalk-cli-setup/blob/102025cb0d1d38e949fde2ca6455764c6f8475fb/scripts/bundled_installer#L10) to refer to `3.9.1`, which _does_ build. You may have to follow pyenv's [docs](https://github.com/pyenv/pyenv/wiki/Common-build-problems) to ensure you have the prerequisite software installed.
2. With a usable Python installed, installing `cryptography` may now fail; see https://github.com/pyca/cryptography/issues/5702. If so, you'll have to build it from source. To do this, follow the instructions [here](https://cryptography.io/en/latest/installation.html#building-cryptography-on-macos) to get prereqs, then finally rerun the `eb` installer script with the flags they prescribe:
```sh
LDFLAGS="-L$(brew --prefix openssl@1.1)/lib" CFLAGS="-I$(brew --prefix openssl@1.1)/include" ./scripts/bundled_installer
```
3. Follow the "Finishing up" instructions from the `eb` installer.

**Comment by prkirby** [comment]
@spencerwilson haha that you for the perfect timing on this, was literally trying to install on a new macbook pro yesterday and ran into this issue. 3.9.1 worked like a charm. Thanks!

**Comment by hishamMounir** [comment]
@spencerwilson Hi, changing the version didn't work with me, now I have this error: python-build: definition not found: 3.9.1

**Comment by prkirby** [comment]
@hishamMounir You have to install python vs 3.9.1, same thing happened to me. You can use pyenv for that, which can be installed with brew

**Comment by spencerwilson** [comment]
@hishamMounir Yep, try installing (or updating, if already installed) pyenv. That error might be symptomatic of pyenv not having an up-to-date list of versions to build (idk if that's a thing; just speculating).

**Comment by hishamMounir** [comment]
I had to install Python manually then, I built cryptgraphy and installed, then I used Homebrew to install the ebcli package instead of building the script, and it worked

**Comment by jpsilvashy** [comment]
Regarding @hishamMounir's approach, that worked for me after installing python with pyenv I did: `brew install awsebcli` and everything worked.
